### PR TITLE
Add query-less merchandisable search preview

### PR DESCRIPTION
**Pull request by pepopowitz**
Addresses https://artsyproduct.atlassian.net/browse/DISCO-725.

A collaboration 👯‍♂️with @jonallured! 

This adds an auto-suggest preview for merchandisable artworks with no suggestion selected. It looks like this: 

image

1. This doesn't match the zeplin prototypes exactly. We kept the visual faffing-about to a minimum while we waited for them to be completed. We can circle back to match them if/when they are done!

2. There are no tests, but there isn't a whole lot of trickery in these components. We can add a snapshot if we feel it's helpful, but probably won't unless someone asks for it.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 81c12a17437e0ead6565399f377646381df25e69

https://deploy-preview-2006--artsy-reaction.netlify.com

File: src/Components/Search/Previews/Grids/MerchandisableArtworks.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            environment={relayEnvironment}
+            variables={{}}
+            query={graphql`
+              query MerchandisableArtworksPreviewQuery {
+                viewer {
+                  ...MerchandisableArtworks_viewer
+                }
+              }
+            `}
+            render={renderWithLoadProgress(
```
Comment (mzikherman) [inline src/Components/Search/Previews/Grids/MerchandisableArtworks.tsx]: Nice use of the helper method.

**Comment by mzikherman** [comment]
Looks perfect to me!

**Comment by artsyit** [comment]
:rocket: PR was released in v12.1.31 :rocket:

**Pull request merged**